common/errors.js

```diff
@@ -184,10 +184,10 @@ class UnprocessableEntity extends HttpClientError {
 exports.UnprocessableEntity = UnprocessableEntity;
 
 class DeploymentAlreadyLocked extends UnprocessableEntity {
-  constructor(deploymentName, lockInfo, lockMessage) {
-    let message = `Deployment ${deploymentName} ${CONST.OPERATION_TYPE.LOCK}`;
+  constructor(instanceId, lockInfo, lockMessage) {
```

Comment:
This error is used for showing deployment being locked, so at some other place in code it would be still sending in deployment Name, so please check that code as well.